alliance/apps/shared/models/team_project.py

```diff
@@ -12,7 +12,7 @@ class TeamProject(models.Model):
     claim_backlog = models.CharField(max_length=100, blank=True, null=True)
 
     def __unicode__(self):
-        return self.team, self.project
+        return '{}, {}'.format(self.team, self.project)
```

Comment:
I believe we're using python2, in which case this string needs a `u` in front of it:
```
return u'{}, {}'.format(self.team, self.project)
```
otherwise if the team name or project name have unicode characters you'll get an error here when python tries to use an ascii codec to decode the string.